articles/extensions/delegated-admin/hooks.md

```diff
@@ -191,24 +191,37 @@ Supported action names:
  - `unblock:user`
  - `send:verification-email`
 
-#### Create Hook
+#### Write Hook
 
-Whenever new users are created you'll want these users to be assigned to the group/department/vendor/... of the current user. This is what the **Create Hook** allows you to configure.
+Whenever new users are created you'll want these users to be assigned to the group/department/vendor/... of the current user. This is what the **Write Hook** allows you to configure.
```

Comment:
It's not clear that there hook is run also during update. Even if the contract includes `method` it would be better to clearly mention when this hook is executed

Reply:
Fixed

Reply:
I'm a bit confused by 
> assigned to the group/department/vendor/... of the current user

Does this mean that if I'm creating a user and I'm part of the IT group, then the new user is part of the IT group, too?

If so, then maybe something like this?

> The **Write Hook** allows you to automatically assign newly-created users to the appropriate group, department, or vendor. Generally, this is the same category to which you belong.